variables.tf

```
@@ -39,9 +39,15 @@ variable "config" {
     frameworks                 = optional(list(string), [])
     log_level                  = optional(string)
     report_suppressed_findings = optional(bool, false)
-    zone_name                  = string
+    single_account_id          = optional(string)
+    zone_name                  = optional(string)
   })
   description = "Map containing labeler configuration options"
+  validation {
+    condition     = var.config.zone_name != "" || var.config.single_account_id != ""
+    error_message = "Either zone_name or single_account_id is required"
+  }
+
```

```suggestion

  validation {
    condition     = var.config.zone_name != "" || var.config.single_account_id != ""
    error_message = "Either zone_name or single_account_id is required"
  }
```

Please fix the formatting here.